projects/bp-gallery/src/hooks/get-pictures.hook.ts

```diff
@@ -52,6 +54,7 @@ const useGetPictures = (
       },
       sortBy,
     },
+    fetchPolicy,
```

Comment:
Maybe add `fetchPolicy` to the other query below, too? Also, if you include the `getMostLikedPicturesQuery` here, add it there, too.